tests/framework/installer/ceph_manifests.go

```diff
@@ -270,6 +270,10 @@ spec:
                   type: integer
                 activeStandby:
                   type: boolean
+								activeStandbyCount:
```

Comment:
your editor likely inserted tabs instead of spaces here, if you could check a few other places in this PR for whitespace as well